### DISCUSSION TITLE
Does RisingWave have a RoadMap?

### DISCUSSION THREAD

**Discussion by D2Lark** [risingwavelabs/risingwave#1771] [Q&A] [answered]
I have only found a few relevant issues to manage the current progress of the project.

https://github.com/singularity-data/risingwave/pull/558
https://github.com/singularity-data/risingwave/pull/1499
https://github.com/singularity-data/risingwave/pull/1678

Given that the RisingLight project currently has 140k lines of code, I am a bit scared to get involved in the project, although I would like to. It would be much less difficult to get started if a roadmap for future development was provided so that developers could work on one or two modules rather than the whole project.

**Comment by yingjunwu** [comment, marked as answer]
Thanks for your interest in RisingWave! We are working on the roadmap and will release later this month. Hope my answer helps.

**Reply by D2Lark** [reply]
thanks！
